cross/x11proto/Makefile

```diff
@@ -0,0 +1,25 @@
+PKG_NAME = xorgproto
```

Comment:
Usually we try to keep the folder name and the PKG_NAME the same. Please follow this rule/guideline for constancy.